app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int, **kwargs) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        new_person = Person(name, age)
+        person_list.append(new_person)
+
+    for person in people:
+        name = person["name"]
+        wife = person.get("wife")
+        husband = person.get("husband")
+
+        if wife is not None:
```

Comment:
`is not None` redundant here, since in your case if statement will be True if key exists

Reply:
done